contracts/truefi2/CreditModel.sol

```diff
@@ -62,6 +62,7 @@ contract CreditModel is ICreditModel, UpgradeableClaimable {
         // adjust for pool value (basis precision)
         uint16 poolValueLimitCoefficient;
         uint16 ltvRatio;
+        uint16 effectiveScorePower;
```

Comment:
Now that I reflect on this some more, I've realized that `ltvRatio` and `effectiveScorePower` don't belong in the `BorrowLimitConfig`.
- `ltvRatio` will affect _both_ borrow limits and interest rates, so it doesn't make sense to update it with `BorrowLimitConfig` when we just want to alter how conservative we're being with borrower staking.
- `effectiveScorePower` _only_ affects interest rates, not borrow limits, so it definitely doesn't belong in `BorrowLimitConfig`.

These should be in a separate config (maybe named `BorrowerStakingConfig`?), since we'll most likely modify these values simultaneously.